app/controllers/users_controller.rb

```diff
@@ -0,0 +1,67 @@
+
+class UsersController < ApplicationController
+  skip_before_action :require_login,only: [:feed]
+
+  def show
+    @creators = current_user.creators
```

Comment:
When you use `current_user` here, you're actually calling the `current_user` method that's in the application controller. Since `current_user` returns `@current_user`, this works, but you're calling the method twice (once in the `before_action`, and once here). Instead, you should be using `@current_user`.